src/protagonist/API.Tests/Features/DeliveryChannels/Validation/HydraDeliveryChannelPolicyValidatorTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using API.Features.DeliveryChannels.Validation;
+using DLCS.HydraModel;
+using FluentValidation.TestHelper;
+
+namespace API.Tests.Features.DeliveryChannelPolicies.Validation;
+
+public class HydraDeliveryChannelPolicyValidatorTests
+{
+    private readonly HydraDeliveryChannelPolicyValidator sut;
+    
+    public HydraDeliveryChannelPolicyValidatorTests()
+    {
+        sut = new HydraDeliveryChannelPolicyValidator();
+    }
+    
+    [Fact]
+    public void NewDeliveryChannelPolicy_CannotHave_AssetId()
+    {
+        var policy = new DeliveryChannelPolicy()
+        {
+            Id = "foo",
+        };
+        var result = sut.TestValidate(policy);
+        result.ShouldHaveValidationErrorFor(p => p.Id);
+    }
+    
+    [Fact]
+    public void NewDeliveryChannelPolicy_CannotHave_CustomerId()
+    {
+        var policy = new DeliveryChannelPolicy()
+        {
+            CustomerId = 1,
+        };
+        var result = sut.TestValidate(policy);
+        result.ShouldHaveValidationErrorFor(p => p.CustomerId);
+    }
+    
+    [Fact]
+    public void NewDeliveryChannelPolicy_CannotHave_Channel()
+    {
+        var policy = new DeliveryChannelPolicy()
+        {
+            Channel = "iif-img"
```

Comment:
`iiif-img`

Reply:
Fixed in https://github.com/dlcs/protagonist/pull/739/commits/986b59c2887ed1f7288c3b50239114bcdc627fa4